index.js

```diff
@@ -193,6 +196,21 @@ function onchange (match, command, rawArgs, opts = {}) {
   watcher.on('error', (error) => log(`watcher error: ${error}`))
 }
 
+function getIgnoreMergedFromIgnoreFile(exclude = [], ignorePath) {
+    if(existsSync(ignorePath)) {
+      const ignoreFileString = readFileSync(ignorePath).toString('utf-8')
+      const ignoreFileArray = ignoreFileString.replace(/^#[^\n]+\n/gm, '').split(/\n/)
```

Comment:
This wouldn't handle Windows newline characters `\r\n`, is there a module that does this already and has tests?

Reply:
Fixed